README.md

```diff
@@ -288,9 +288,9 @@ If `WARM_IP_TARGET` is set, then this environment variable is ignored and the `W
 
 Type: Integer
 
-Default: None
+Default: `1`
```

Comment:
So the default here is still `None`, as when set, the WARM/MINIMUM IP targets take precedence over the WARM_ENI_TARGET. The default when `MINIMUM_IP_TARGET` is set is 1